package.json

```diff
@@ -31,7 +31,7 @@
   "homepage": "https://github.com/alpheios-project/wordsvc#readme",
   "devDependencies": {
     "alpheios-client-adapters": "github:alpheios-project/client-adapters",
-    "alpheios-components": "github:alpheios-project/components",
+    "alpheios-components": "file://../components",
```

Comment:
I think this needs to be changed

Reply:
oops yes, sorry will fix that before merging.